### ISSUE TITLE
Response 500s

### ISSUE THREAD

**Issue by gzuuus** [PlebeianTech/plebeian.market#474]
There are various users reporting response 500 in different parts of the platform, creating products, stalls, updating order status

**Comment by Aslemammad** [comment]
i think we need more reproduction steps for these!

**Comment by gzuuus** [comment]
Yes, let's keep an eye on the 500s, some of them are already gone.
